tripaldocker/Dockerfile-php8.2-pgsql13

```diff
@@ -1,13 +1,13 @@
 FROM php:8.2-apache-bullseye
 
-ARG drupalversion='9.4.x-dev'
+ARG drupalversion='10.1.0'
```

Comment:
There are actually multiple versions of Drupal 10.1 and 10.1.0 specifically has a security advisory on it. Since this version is used by composer we can use a tilde here to specify, any version between 10.1.0 and up to but not including 10.2.0. This ensures we get the most recent stable release of Drupal 10.1

```suggestion
ARG drupalversion='~10.1.0'
```

Reply:
@laceysanderson Thanks for the suggestion and teaching me something new! That's really cool.